mono/metadata/object-offsets.h

```diff
@@ -136,7 +136,8 @@ DECL_OFFSET(MonoProfilerCallContext, method)
 DECL_OFFSET(MonoProfilerCallContext, return_value)
 DECL_OFFSET(MonoProfilerCallContext, args)
 
-DECL_OFFSET(MonoError, init)
+//FIXME
+//DECL_OFFSET(MonoError, init)
```

Comment:
Could be `DECL_OFFSET(MonoErrorInternal, init)` and then `MONO_STRUCT_OFFSET (MonoErrorInternal, init)` at the use site in `decompose.c`, right?

Reply:
Again I was being lazy. I will try that.
I'd actually like to remove this, but I have some inexplicable wasm-only error:
https://github.com/mono/mono/pull/15894